change/react-native-windows-2020-06-01-15-09-20-pull_request.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
This should be "prerelease". We only want "none" in cases where the change doesn't affect the published package.